src/helpers.js

```diff
@@ -0,0 +1,11 @@
+var config = require("../tmp/config.json");
+var CAPI_API_KEY = config.capi_key;
+
+exports.capi_query = function(endpoint,filter, q) {
```

Comment:
I believe CamelCase is preferred in node.js: https://github.com/felixge/node-style-guide